src/includes/constants.php

```diff
@@ -58,39 +57,39 @@
 }
 
 // Allow a first class env var to be set that disables .env file loading
-if (!getenv('SS_IGNORE_DOT_ENV')) {
+if (!Environment::getEnv('SS_IGNORE_DOT_ENV')) {
```

Comment:
Just revisiting what @sminnee said - this should only ever be set by a global bonefido environment variable that won't change from execution to execution. As the comment says "Allow a first class env var to be set that disables .env file loading" but this allows second-class / psuedo environment variables to manipulate/unset this value which is the opposite of what's intended here.

Reply:
> but this allows second-class / psuedo environment variables to manipulate/unset this value which is the opposite of what's intended here.

`getenv` allows exactly the same if you use `putenv`, so what's the introduced risk that didn't exist before?